# LAZER_DISTANCE message data

Header header

float32 min_distance
float32 angle
float32 lazer_x
float32 lazer_y
